org-roam-bibtex-note-actions.el

```diff
@@ -200,6 +200,14 @@ that can be customized.  Additionally, user actions can be set in
   (interactive)
   (let ((non-default-frontends (list 'hydra 'ido 'ivy 'helm))
         (citekey (cdr (assoc "ROAM_KEY" (org-roam--extract-global-props '("ROAM_KEY"))))))
+    ;; remove format from citekey
+    (when org-roam-bibtex-citekey-format
+      (string-match "\\(.*\\)%s\\(.*\\)" org-roam-bibtex-citekey-format)
+      (let ((beg (match-end 1))
+            (end (+ (length citekey)
+                    (- (match-beginning 2)
+                       (length org-roam-bibtex-citekey-format)))))
+        (setq citekey (substring citekey beg end))))
```

Comment:
Could we turn this into a function?  We might want to be using it elsewhere in the future.

Reply:
Thanks, done in 656e83197a5affebf60964af3ea936d2faecda79!